lib/vector-api-client/src/gql/components.rs

```diff
@@ -36,3 +78,43 @@ impl std::fmt::Display for components_query::ComponentsQueryComponentsOn {
         write!(f, "{}", res)
     }
 }
+
+impl std::fmt::Display
+    for component_added_subscription::ComponentAddedSubscriptionComponentAddedOn
+{
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        let res = match self {
+            component_added_subscription::ComponentAddedSubscriptionComponentAddedOn::Source => {
```

Comment:
Minor nit that you can make this quite a bit less noisy with `use component_added_subscription::ComponentAddedSubscriptionComponentAddedOn` in this file.

Reply:
Unfortunately I don't think that will work, due to `component_added_subscription` and ilk being auto-generated in this same file.